ova/generate_ova.sh

```diff
@@ -55,163 +64,191 @@ clean() {
 
     cd ${scriptpath}
     vagrant destroy -f
-    rm -f ${OVA_VM} ${OVF_VM} ${OVA_VMDK} ${OVA_FIXED} Config_files/filebeat.yml
-
+    OVA_VMDK="wazuh-${OVA_VERSION}-disk001.vmdk"
+    rm -f ${OVA_VM} ${OVF_VM} ${OVA_VMDK} ${OVA_FIXED}
+    
     exit ${exit_code}
 }
 
 build_ova() {
 
+    # Files
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/6fb50e3584df557334da525324a73a5c32ae0551